pkgs/top-level/ocaml-packages.nix

```diff
@@ -762,6 +762,8 @@ let
 
     mparser =  callPackage ../development/ocaml-modules/mparser { };
 
+    mparser-pcre =  callPackage ../development/ocaml-modules/mparser-pcre { };
```

Comment:
Consider moving `mparser-pcre` to `../development/ocaml-modules/mparser/pcre.nix`. I've started to establish this convention for multiple ocaml packages which are built from the same source archive and share a common prefix.